base/requirements.txt

```
@@ -1,2 +1,2 @@
 python-dateutil==2.8.0
-git+https://github.com/confluentinc/confluent-docker-utils@v0.0.38
+git+https://github.com/confluentinc/confluent-docker-utils@v0.0.39
```

```suggestion
git+https://github.com/confluentinc/confluent-docker-utils@niteshmor-jinja-upgrade
```

You should also be able to test against https://github.com/confluentinc/confluent-docker-utils/pull/23 here by using the branch for the PR `niteshmor-jinja-upgrade` here.